src/components/Controllers/View/store.ts

```diff
@@ -32,17 +39,89 @@ export function createStore(props: SqlProps) {
     loadFields: async () => {
       const { client } = get()
       const { items } = await client.fieldList()
+      const tables: string[] = []
+      for (const item of items) {
+        if (tables.indexOf(item.tableName) < 0) {
+          tables.push(item.tableName)
+        }
+      }
       set({ fields: items })
+      set({ tables: tables })
     },
     makeQuery: async () => {
       const { client, view } = get()
       if (!view) return
-      const { table } = await client.tableQuery({ query: view.query })
-      set({ table })
+      const parser = new Parser()
```

Comment:
We need to split:
- query validation -> Editors/View (as it's shown now for example of the editor loosing focus we do query validation)
- result validation (from server) -> Controllers/View (showing the errors instead of the table)